### ISSUE TITLE
Date range

### ISSUE THREAD

**Issue by lastmike** [ppalucha/ksar2#17]
Thank you for ksar2.
In previous ksar version, we can select a date range.
Is-it possible to re-introduce this functionnality?

Thank you.

**Comment by ppalucha** [comment]
Hi,
I would need some more information on what ksar version you refer and what is exactly that functionality you would like to have. For sure I did not remove any valid functionality ;-)
Please note you can zoom in and out the graph.

**Comment by ubellavance** [comment]
Yes, I know that we can zoom in and out, but with the original kSar, we can use the -s and -e command-line options to only pull the data for a specific time period.  When we use -s and -e with ksar2, they are simply ignored.  It can be useful mostly to compare stats for 2 servers or 2 days for the (exact) same time period.  With the original kSar, it could also be useful when there are so many stats. that kSar kind of don't work... I've seen this with a server that hat many LUNs.
I think that there is also a way to show a time period shorter than what sar sends, but honestly, I'd rather use the -e and -s because the scaling wasn't good.

**Comment by Decstasy** [comment]
@ubellavance You can use an old kSAR Version to do this once. 
Since old Version sometimes hate it to parse specific datasets of your report, I made a little script which removes the exception causing datasets from your file. You can put it in the old kSar after applying the script without problems. https://gist.github.com/Decstasy/98c0f9b5bf987b88afe630972f91fb84

**Comment by ubellavance** [comment]
@Decstasy I don't really understand what you're talking about... sorry.  Maybe wrong issue?

**Comment by lastmike** [comment]
![blog-ksar0-600x480](https://user-images.githubusercontent.com/35777639/45077777-5d5ed080-b0ee-11e8-8908-70a76118e38d.png)

@ppalucha : as you can see in attached picture, in ksar an entry is called "Select time range..". This entry is available if you previously call "Load from text file".

Our use case is: 
We launch a bench of our software and we want extract ksar graph for the bench time range.
 
We need ksar2 because we extract SNMP informations and ksar failed to open this kind of informations.

Thank you.
